frontend/Dockerfile

```
@@ -9,23 +9,25 @@ RUN apt-get update \
     && apt-get install -y --no-install-recommends ca-certificates curl \
     && rm -rf /var/lib/apt/lists
 
-COPY elm/elm_arm64 /usr/local/bin/elm
-
 WORKDIR /usr/src/app
 
 COPY frontend/elm.json ./
 
-# https://github.com/elm/compiler/blob/24d3a89469e75cf7aa579442ecaf5ddfdd192ab2/installers/linux/README.md
-RUN if [ "${TARGETPLATFORM}" != "linux/arm64" ]; then \
-        curl -L https://github.com/mikefarah/yq/releases/latest/download/yq_linux_amd64 -o /usr/bin/yq; \
-        chmod +x /usr/bin/yq; \
-        elm_version="$(yq '."elm-version"' elm.json)"; \
-        curl -L -o elm.gz "https://github.com/elm/compiler/releases/download/${elm_version}/binary-for-linux-64-bit.gz"; \
-        gunzip elm.gz; \
-        chmod +x elm; \
-        mv elm /usr/local/bin/; \
-        rm /usr/bin/yq; \
-    fi
+RUN curl -L https://github.com/mikefarah/yq/releases/latest/download/yq_linux_amd64 -o /usr/bin/yq \
+        && chmod +x /usr/bin/yq; \
+    elm_version="$(yq -r '."elm-version"' elm.json)"; \
+    case ${TARGETPLATFORM} in \
```

```suggestion
    && chmod +x /usr/bin/yq; \
    && elm_version="$(yq -r '."elm-version"' elm.json)" \
    && case ${TARGETPLATFORM} in \
```

インデントがちょっとずれてるのと `&&` で繋げるのが良さそう。